### ISSUE TITLE
Thank you!

### ISSUE THREAD

**Issue by zWhdmB5T** [osmandapp/OsmAnd#18834]
Thank you!

Thank you for your project and software!

Thank you for maintenance in the past and in the future!

Kind regards and season's greetings!

**Comment by itzikiap** [comment]
Yes!
Also thank you for such a robust and feature rich app!

**Comment by albansuser** [comment]
I agree. To the development and support team, thank you and best wishes for the new year.
